IronGram/src/main/java/com/theironyard/entities/Photo.java

```diff
@@ -13,13 +16,19 @@
     private int id;
 
     @ManyToOne
-    User sender;
+    private User sender;
 
     @ManyToOne
-    User recipient;
+    private User recipient;
+
+    @Column(nullable = false)
+    private String filename;
 
     @Column(nullable = false)
-    String filename;
+    private boolean photoPublic;
+
+    @Column(nullable = false)
+    private long lifeDuration;
```

Comment:
There needs to be some kind of commenting to indicate what is being stored.  Is this seconds? milliseconds? minutes?